app/build.gradle

```diff
@@ -387,4 +409,4 @@ task printGeckoviewVersions {
 
 // Normally this should use the same version as the glean dependency. But since we are currently using AC snapshots we
 // can't reference a git tag with a specific version here. So we are just using "master" and hoping for the best.
-apply from: 'https://github.com/mozilla-mobile/android-components/raw/master/components/service/glean/scripts/sdk_generator.gradle'
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/master/components/service/glean/scripts/sdk_generator.gradle'
```

Comment:
Nit: EOL

Reply:
Is this something that should be addressed? I think that *nix prefers there to be an EOL at EOF, but Windows doesn't care. See [this point](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

Mildly-relevant side-note: some of my tools automatically add a line to EOF, and I don't want to hunt them down and disable 'em :P
Are you ok with leaving this in?